src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/Target_ARM/ARMEmitter.cs

```diff
@@ -32,12 +61,44 @@ public void EmitPUSH(Register reg)
             Builder.EmitShort((short)(0x0d04 + ((byte)reg << 12)));
         }
 
+        // pop reg
+        public void EmitPOP(Register reg)
+        {
+            Builder.EmitByte(0x5d);
+            Builder.EmitByte(0xf8);
+            Builder.EmitShort((short)(0x0b04 + ((byte)reg << 12)));
+        }
+
         // mov reg, reg
+        // reg range: 0-7
```

Comment:
It's thumb1, it's not mov.w, all registers have only 3 bits. Range r0-r7, exception is PC register, it's allowed too. 
But I decided not to write about it, maybe in error handling later I'll add it too.